.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,199 @@
+name: Benchmark Pull Request
+on:
+  pull_request:
+  repository_dispatch:
+    types: [ reporters-db-pr ]
+
+env:
+  main: "$(/usr/bin/git log -1 --format='%H')"
+
+jobs:
+  benchmark:
+    name: PR comment
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      #----------------------------------------------
+      #       check-out repo and set-up python
+      #----------------------------------------------
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Set up python
+        id: setup-python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      #----------------------------------------------
+      #  -----  install & configure poetry  -----
+      #----------------------------------------------
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+        with:
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+          installer-parallel: true
+          version: 1.2.0rc2
+      #----------------------------------------------
+      #       load cached venv if cache exists
+      #----------------------------------------------
+      - name: Load cached venv
+        id: cached-poetry-dependencies
+        uses: actions/cache@v2
+        with:
+          path: .venv
+          key: venv-${{ runner.os }}-${{ steps.setup-python.outputs.python-version }}-${{ hashFiles('**/poetry.lock') }}
+      #----------------------------------------------
+      # install dependencies if cache does not exist
+      #----------------------------------------------
+      - name: Install dependencies
+        if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
+        run: poetry install --with benchmark --without dev
+
+      - name: Setup variables I
+        id: branch1
+        run: |
+          echo ${{ github.event.issue.pull_request }}
+          echo "::set-output name=filepath::benchmark/${{ env.main }}.json"
+          echo "::set-output name=hash::${{ env.main }}"
+
+      - name: Download Testing File
+        run: |
+          curl https://storage.courtlistener.com/bulk-data/eyecite/tests/one-percent.csv.bz2 --output benchmark/bulk-file.csv.bz2
```

Comment:
We need a comment here or somewhere about how this file was made so we can remake it. I think I shared the details with you on Slack at some point?

Reply:
and yes for the record this is what we used.  
```
root@maintenance:/opt/courtlistener# PGPASSWORD=$DB_PASSWORD psql       --command         'set statement_timeout to 0;
           COPY (SELECT id, plain_text, html, html_lawbox, html_columbia, html_anon_2020, xml_harvard FROM search_opinion TABLESAMPLE BERNOULLI (0.1)) TO STDOUT WITH (FORMAT csv, ENCODING utf8, HEADER)'      --host $DB_HOST         --username $DB_USER     --dbname courtlistener  |       bzip2 |         aws s3 cp - s3://com-courtlistener-storage/bulk-data/eyecite/tests/ten-percent.csv.bz2 --acl public-read
root@maintenance:/opt/courtlistener# PGPASSWORD=$DB_PASSWORD psql       --command         'set statement_timeout to 0;
           COPY (SELECT id, plain_text, html, html_lawbox, html_columbia, html_anon_2020, xml_harvard FROM search_opinion TABLESAMPLE BERNOULLI (0.01)) TO STDOUT WITH (FORMAT csv, ENCODING utf8, HEADER)'     --host $DB_HOST         --username $DB_USER     --dbname courtlistener  |       bzip2 |         aws s3 cp - s3://com-courtlistener-storage/bulk-data/eyecite/tests/one-percent.csv.bz2 --acl public-read```

Reply:
Sure, or just put it here as a comment if that's easier. That command looks right to me.

Reply:
added